packages/mask/src/plugins/NextID/SNSAdaptor/index.tsx

```diff
@@ -29,7 +29,8 @@ const sns: Plugin.SNSAdaptor.Definition = {
         )
     },
     PostActions() {
-        if (!Flags.next_id_tip_enabled) return null
+        const pluginId = usePluginIDContext()
+        if (!Flags.next_id_tip_enabled || pluginId !== NetworkPluginID.PLUGIN_EVM) return null
```

Comment:
Prefer to use `enableRequirement`.

Reply:
> Prefer to use `enableRequirement`.

Then the whole `NextID` plugin will be disabled.

Reply:
That will be fine. NextID only supports EVM.